airflow/utils/db.py

```diff
@@ -300,6 +300,17 @@ def initdb():
         Connection(
             conn_id='opsgenie_default', conn_type='http',
             host='', password=''))
+    merge_conn(
+        Connection(
+            conn_id='drill_default', conn_type=''
+        )
+    )
+    merge_conn(
+        Connection(
+            conn_id='drill_default', conn_type='drill+sadrill',
+            host='localhost', port=8047
+        )
+    )
```

Comment:
Why two connections with the same id are merged here

Reply:
Ah, Sorry. I will fix it soon.